jujugui/static/gui/src/app/init/hotkeys.js

```diff
@@ -144,7 +130,7 @@ const activate = context => {
         }
         if (spec.focus) { target.focus(); }
       }
-      if (spec.callback) { spec.callback.call(context, evt, target); }
+      if (spec.callback) { spec.callback.call(this, evt, target); }
```

Comment:
In this case aren't you calling the callback in the `window` context? is that intentional?

Reply:
I should have just removed the call.